### ISSUE TITLE
Disable all linters (by glob or merge_with_default option)

### ISSUE THREAD

**Issue by sgregson** [brigade/scss-lint#219]
Is it possible to disable all linters with a wildcard (without disabling them by name)? My intention was to create a reset configuration so that any new/changed linters introduced in future versions would be disabled by default until explicitly enabled in my configuration. This also be a flag that blocks the default config.

**(option A)** reset by glob

``` yml
# .scss-lint-reset.yml
linters:*:
  enabled: false
```

``` yml
# .scss-lint.yml
inherit_from: '.scss-lint-reset.yml'

linters:
  BorderZero:
    enabled: true
    severity: warning
  # explicitly enabled lints
  ZeroUnit:
    enabled:true
    severity: error
```

**(option B)** set the merge_with_default option in the config file (tying into [config.rb](http://www.github.com/causes/scss-lint/blob/2fae846/lib/scss_lint/config.rb#L18)):

``` yml
# .scss-lint.yml
merge_with_default: false

linters:
  BorderZero:
    enabled: true
    severity: warning
  # explicitly enabled lints
  ZeroUnit:
    enabled:true
    severity: error
```
